bin/pycbc_make_sky_grid

```diff
@@ -165,23 +165,20 @@ spher = cart_to_spher(rota)
 
 # Calculate the time delays between the Earth center
 # and each detector for each sky point
-time_delays = [
-    [
-        detectors[i].time_delay_from_earth_center(
-            spher[j][0], spher[j][1], args.trigger_time
+sky_grid = SkyGrid(
+        spher[:, 0], spher[:, 1], args.instruments, args.trigger_time
         )
-        for j in range(len(spher))
-    ]
-    for i in range(len(detectors))
-]
+time_delays = sky_grid.calculate_time_delays()
+
+extra_attributes = {
+        'trigger_ra': args.ra,
+        'trigger_dec': args.dec,
+        'sky_error': args.sky_error,
+        'timing_uncertainty': args.timing_uncertainty
+        }
+
+extra_datasets = {
+        'time_delays': time_delays
+        }
```

Comment:
I realized this will probably not work, because `time_delays` is now a dict, while in the previous version it was a list of lists of floats. I doubt h5py lets you store a dict as a dataset.

In the interest of simplicity, I propose to simply not store the time delays in the output file for now. Nothing uses them at the moment. If needed, we will revisit this later as an optimization change.

Please also remove the comment on lines 166-167 for consistency.